CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+
+- When we're exceeding the maximum number of retries, the exception object now contains a string message instead of the last unsuccessful request object.
```

Comment:
```suggestion
- When we're exceeding the maximum number of retries, the exception type is now `RecoverableServerError` and the object contains a string message instead of the last unsuccessful request object.
```

Reply:
No, we always returned `RecoverableServerError` in this case. The only difference is that now the args of the exception object are different.